app/build.gradle

```diff
@@ -7,8 +7,8 @@ android {
         applicationId "com.intervaltimer.google"
         minSdkVersion 21
         targetSdkVersion 29
-        versionCode 4
-        versionName "1.2"
+        versionCode 5
```

Comment:
Why is this change needed?

Consider to think about "tracking approaches" for changes which are required for the final pull request

Reply:
The app was internally published on Google Play twice for testing purposes. However, to publish a new release, the version code has to be changed.

The changes that aren't necessary for the final pull request are mostly in the separate commit (Package renaming). But I agree that some of the might be potentially confused together. What kind of tracking do you have in mind?

Reply:
> What kind of tracking do you have in mind?

I think there are a bunch of potential approaches, e.g., marking it in the cl that it's only needed for testing, adding a comment that it's only needed for testing, having a separate list with changes what's only required for testing (but the first two approaches have the benefit that they make it explicit for the reviewers/us)... but. I don't have a strong preference and I'm not sure if it is necessary...